packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -23,7 +23,7 @@ export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any')
     'commentToFulfiller,drug:(uuid,display,strength,dosageForm:(display,uuid),concept),dose,doseUnits:ref,' +
     'frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,' +
     'duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)';
-  const ordersUrl = `${restBaseUrl}/order?patient=${patientUuid}&careSetting=${careSettingUuid}&status=${status}&orderType=${drugOrderTypeUUID}&v=${customRepresentation}`;
+  const ordersUrl = `${restBaseUrl}/order?patient=${patientUuid}&careSetting=${careSettingUuid}&orderTypes=${drugOrderTypeUUID}&v=${customRepresentation}`;
```

Comment:
Is it right for the URL param here to change from `orderType={drugOrderTypeUUID}` to `orderTypes={drugOrderTypeUUID}`, @ibacher @dkayiwa?

Reply:
Missed that... `orderType`... `orderTypes` won't do anything.

Reply:
@ibacher,  @denniskigen When I switch to orderType, we will primarily be requesting orders of a specific type (active medications) based on the UUID provided in drugOrderTypeUUID, which excludes other order types such as discontinued or past orders. However, the orderTypes parameter allows querying multiple types of orders (e.g., active, discontinued, or past orders). Since we are retrieving past medications, the API may expect multiple orderType values, such as both active and discontinued orders, with the plural form likely indicating an array of possible order types.  I will therefore request we user `orderTypes  ` Please feel free to correct me if necessary.

Reply:
Active / Discontinue is separate from order type. `orderType=${drugOrderTypeUUID}` restricts things to drug orders, whether active or discontinued. There is no `orderTypes` parameter ([here is the code that parses it](https://github.com/openmrs/openmrs-module-webservices.rest/blob/55682783dcae10b9bca9616c8e50bf886ef8b864/omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java#L218)), so removing this now doesn't correctly filter things to just drug orders.

Actually [looking at the code](https://github.com/openmrs/openmrs-module-webservices.rest/blob/2bb38f2108b19497d78a877f885d6f2a1bc4683f/omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderUtil.java#L42) removing the `status` parameter is also the wrong thing. It should actually be set to "any".